packages/functional-tests/src/tests/user-mask-test.ts

```diff
@@ -73,13 +78,31 @@ export default class UserMaskTest extends Test {
 			}
 		});
 
+		MRE.Actor.Create(this.app.context, {
+			actor: {
+				name: 'notRedIcon',
+				parentId: root.id,
+				appearance: {
+					enabled: new MRE.InvertedGroupMask(this.app.context, ['red', 'default']),
```

Comment:
>InvertedGroupMask [](start = 22, length = 17)

Does this mean not in the default group as well?  Or is every user alway in the default group?

Reply:
Not in the default group, correct.

Reply:
To clarify: every who has no other groups assigned is in the "default" group. So if "default" is in an inverted group mask, then those users will *not* see the given object.

---
In reply to: [437673473](https://github.com/microsoft/mixed-reality-extension-sdk/pull/608#discussion_r437673473) [](ancestors = 437673473)

Reply:
So then it is possible to use the default group for filtering now as well since you can set and object for the "not default group"?

---
In reply to: [437683618](https://github.com/microsoft/mixed-reality-extension-sdk/pull/608#discussion_r437683618) [](ancestors = 437683618,437673473)

Reply:
You could always use the default group for filtering. Remember, the default group is not all users, it's only users with no other groups assigned. Just now you can apply inverted semantics. It's just like any other group.

---
In reply to: [437688309](https://github.com/microsoft/mixed-reality-extension-sdk/pull/608#discussion_r437688309) [](ancestors = 437688309,437683618,437673473)